lang/py/test/test_datafile_interop.py

```diff
@@ -35,8 +35,11 @@ def test_interop(self):
       reader = open(os.path.join('@INTEROP_DATA_DIR@', f), 'rb')
       datum_reader = io.DatumReader()
       dfr = datafile.DataFileReader(reader, datum_reader)
+      i = 0
       for datum in dfr:
```

Comment:
Consider

```python
i = 0
for i, datum in enumerate(dfr):
  assert datum is not None
assert i > 0
```

Reply:
Or maybe even just

```python
assert next(dfr) is not None  # will raise StopIteration if the DataFileReader is completely empty
assert not any(datum is None for datum in dfr)
```

Reply:
Thanks for the comment @kojiromike! I prefer the first one because it's easy to understand at first glance and is consistent with other languages' test codes. Will update the PR.

Reply:
@sekikn Hmm, I think the first one I wrote has a mistake. It should be

```python
i = 0
for i, datum in enumerate(dfr, 1):
  assert datum is not None
assert i > 0
```

The previous implementation will still set i=0 on the first pass.

Reply:
Oh you're right, just updated the PR. Thanks!